README.md

```diff
@@ -74,11 +74,12 @@ corresponds to a `deps/` path. Two types of deps are supported, `:git` and
 `:git` fetches given rev from `HEAD`. It doesn't support other branches just
 yet, see [NixOS/nix#2409](https://github.com/NixOS/nix/pull/2409).
 
-`:hexpm` fetches package from [Hex.pm](https://hex.pm). It comes with a SHA-256
-hash of catenation of Mix archive version, package metadata, and `.tar.gz`
-source code archive. Nix can only carry that catenation into the sandbox, so
-[`binwalk`](https://github.com/ReFirmLabs/binwalk) finds where the source code
-archive starts and extracts that part, ignoring archive version and metadata.
+`:hexpm` fetches package from [Hex.pm](https://hex.pm). It comes with
+a SHA-256 hash of catenation of Mix archive version, package metadata,
```

Comment:
While "catenation" is actually a real word, I had to look it up to confirm it wasn't a typo. Given that this is the same thing as *con*catenation, unless we're talking chemistry, maybe let's use less obscure words.

Reply:
```suggestion
a SHA-256 hash of concatenation of Mix archive version, package metadata,
```

Reply:
Don't forget other instances thereof